bytebase/bytebase.com
# feat: changelog posts update

This pull request brings updates for changelog page:

1. redesign changelog page
2. add changelog single post template

Steps to check:

- open preview [page](https://bytebase-git-changelog-anchors-bytebase-pixelpoint.vercel.app/changelog/), preivew [post](https://bytebase-git-changelog-anchors-bytebase-pixelpoint.vercel.app/changelog/bytebase-2-0-0/)
- make sure that everything looks and feels as expected
